config/config_files/vite.config.ts

```diff
@@ -50,11 +50,19 @@ export default ({ mode }: { mode: string }) => {
               return 'proj4';
             } else if (id.includes('react-beautiful-dnd')) {
               return 'react-beautiful-dnd';
+            } else if (id.includes('mui')) {
+              return 'mui';
             } else if (id.includes('node_modules')) {
               return 'vendor';
             }
           },
         },
+        onwarn(warning, warn) {
```

Comment:
What does thing do/achieve? 🤔

Reply:
https://github.com/TanStack/query/issues/5175

<img width="926" alt="Skjermbilde 2023-08-18 kl  09 15 41" src="https://github.com/equinor/amplify-components/assets/17073708/ba62a7a2-863f-488e-b655-8250d75803c4">

Reply:
React-query gir masse warnings i build på alle steder der useClient er brukt, av typen "Module level directives cause errors when bundled, "use client" in "node_modules/@tanstack/react-query-devtools/build/lib/index.mjs" was ignored."